automated/linux/network-basic/network-basic.sh

```diff
@@ -50,7 +52,15 @@ run "route" "print-routing-tables"
 run "ip link set lo up" "ip-link-loopback-up"
 run "route" "route-dump-after-ip-link-loopback-up"
 run "ip link set ${INTERFACE} up" "ip-link-interface-up"
-run "ip link set ${INTERFACE} down" "ip-link-interface-down"
+# NFS mounted filesystem hangs when interface set down
+# so skip "ip link set <interface> down" when NFS is not false
+
+if [ "${NFS}" = "false" ] || [ "${NFS}" = "False" ]; then
+    run "ip link set ${INTERFACE} down" "ip-link-interface-down"
+else
+    info_msg "Skipping ip link set ${INTERFACE} down on NFS mounted file systems"
+    echo "ip-link-interface-down skip"
```

Comment:
We have "report_skip" from sh-test-lib for skipping tests.

Reply:
oops.
I have added "report_skip" to lib.
thanks for the reminder :-)